code/datums/components/seethrough.dm

```diff
@@ -102,13 +102,14 @@
 	for(var/atom/movable/screen/plane_master/seethrough in our_hud.get_true_plane_masters(SEETHROUGH_PLANE))
 		seethrough.unhide_plane(fool)
 
-	var/image/user_overlay = new(parent)
-	user_overlay.loc = parent
+	var/atom/atom_parent = parent
```

Comment:
Why the typecast?

Reply:
I added a case to SET_PLANE_EXPLICIT that, if the atom (or image/MA) isn't on a turf, instead uses the atom (or image/MA)'s plane directly to determine the plane of the resulting transformation
https://github.com/tgstation/tgstation/blob/5612ea6839409b08abeca5617dd52d6e5f959a47/code/__HELPERS/_planes.dm#L52-L54
Off https://github.com/tgstation/tgstation/commit/9f215c5316e5cfdbedf6a23ff97dfee0e523354b

I could do the typecast inside the macro, but I think the compile time requirement is good, since it ensures people use the macro correctly.

Reply:
Ahhhhh okay that makes sense, I forgot macro's could be doing that